account_banking_pain_base/models/account_payment_order.py

```diff
@@ -378,7 +382,8 @@ def generate_party_block(
         eval_ctx = {'partner_bank': partner_bank}
         party_name = self._prepare_field(
```

Comment:
```python
party_name = self.with_context(
    line=bank_line,
)._prepare_field(
...
```

Reply:
self.with_context will result in a re-browse of the payment order for every line hence performance impact on large payment orders. Adding an extra keyword param does not have this effect (and in the meanwhile I run this patch at a couple of sites for about 9 months without any side-effect, hence it's stable).